tests/test_xns11.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+
+from mikeio.xns11 import (
+    read, Xns11, QueryData, FileNotOpenedError, DataNotFoundInFile
+)
+
+
+def test_query_validate():
+    # Bad topo id type
+    with pytest.raises(TypeError):
+        QueryData(666)
```

Comment:
Is it necessary to raise an exception? Isn't it possible to convert the supplied integer (in this case) to a string for a better user experience?

Reply:
I am not sure if using numeric values for topo-id names is a good practice (same for reach names).